### DISCUSSION TITLE
求助

### DISCUSSION THREAD

**Discussion by linhae6964** [Lunova-Studio/DarkMode2#58] [Q&A]
用的好好的突然就没有了，双击也打不开，卸载了重装也不行，
![a82211c4d3ab463a6ba09bddbf40f5c9](https://github.com/user-attachments/assets/d16d8ee4-4d22-404b-995d-022979db6979)
就提示这个，W11系统，之前用的都好好的，望大神解答，🙏

**Comment by bao-cn** [comment]
加群下载最新版，目前已知的是网易云音乐会占用这个快捷键，试试把网易云退了打开

**Reply by linhae6964** [reply]
麻烦问下群在哪里加啊，谢谢🙏

**Reply by bao-cn** [reply]
> 麻烦问下群在哪里加啊，谢谢🙏
695527760
